IntegrationTests/tests_04_performance/test_01_resources/shared.swift

```diff
@@ -112,10 +112,6 @@ func doRequests(group: EventLoopGroup, number numberOfRequests: Int) throws -> I
             }
         }.bind(to: localhostPickPort).wait()
 
-    defer {
```

Comment:
not sure I follow this change? That just moves the close from the (safer) defer into a less safe place.